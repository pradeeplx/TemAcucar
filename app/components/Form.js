import React from 'react'
import { StatelessForm } from 'react-native-stateless-form'

export default Form = ({style, children}) => (
  <StatelessForm style={style}>
    {children}
  </StatelessForm>
)
