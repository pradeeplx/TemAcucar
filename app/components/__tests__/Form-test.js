import { View } from 'react-native'
import React from 'react'
import Form from '../Form'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Form
      style={{top: 0}}
    >
      <View style={{bottom: 0}}>Foobar</View>
    </Form>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
