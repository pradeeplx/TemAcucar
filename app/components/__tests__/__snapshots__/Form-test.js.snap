exports[`test renders correctly 1`] = `
<ScrollView
  keyboardShouldPersistTaps={false}
  style={
    Array [
      Object {
        "alignSelf": "stretch",
        "flex": 1,
      },
      Object {
        "top": 0,
      },
    ]
  }>
  <View
    style={
      Object {
        "bottom": 0,
      }
    }>
    Foobar
  </View>
</ScrollView>
`;
